template-roles-and-resources.md

```diff
@@ -0,0 +1,167 @@
+- You should be an experienced technical writer with good communication and collaboration skills.
+- You should be familiar with strategies for leading effective writer’s workshops.
+- You should have experience with the template writing process, which is demonstrated by one or more of the following:
+  - Completing a full template set from beginning to end.
+  - Participating in a template-focused working group for at least 2 months.
+- You must volunteer and be endorsed for this role by the templates coordinator, a template mentor, or a member of the project steering committee.
+
+
+## Community reviewers
+
+In addition to the members of template writing working groups, this group of volunteer templateers review templates during the workshop phase.
+
+> :triangular_flag_on_post: **NOTE: At least three community reviews are required for each template, but more are preferred.**
+
+Acting as a community reviewer is a good way for an experienced technical writer to start getting involved with The Good Docs Project and provide a first contribution.
+
+The current volunteer community reviewers are:
+
+- Alyssa Rock
+- Aidan Doherty
+- Anu Ramanathan
+- Cameron Shorter
+- Felicity Brand
+- Your name here! There is no limit to how many can join this group!
+
+
+### Responsibilities
+
+Community reviewers:
+- Respond to requests from the templates coordinator and/or template mentors when a community member has a draft that needs review.
+- Provide meaningful feedback that will help the template produce a high-quality template.
+- Encourage both global-level revisions (on the document as a whole) and local-level revisions (at the paragraph, sentence, or word level) as needed.
+- Strive to give feedback that is constructive and avoid making comments that will increase defensiveness.
+- Be available (asynchronously) to the templateer if they have follow-up questions or concerns about your feedback.
+
+
+### Eligibility
+
+To become a community reviewer:
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should have writing experience or subject matter expertise for the type of document being reviewed.
+- You should be familiar with the [Conventional Comments](https://conventionalcomments.org/) theory of labeling feedback.
+- Let the current templates coordinator or a template mentor know that you want to volunteer as a community reviewer.
+
+
+## Pull request reviewers
+
+This group of experienced templateers review and approve pull requests to the templates repository.
+
+> :triangular_flag_on_post: **NOTE: At least two pull request reviewer approvals are necessary to merge a template into the project.**
+
+The current pull request reviewers are:
+
+- Aidan Doherty
+- Alyssa Rock
+- Cameron Shorter

```

Comment:
Should Aaron be added to this list? Since he is templates coordinator?

Reply:
I don't think Aaron has plans to lead a template focused working group.